docroot/themes/custom/yukonca_glider/tailwind.config.js

```diff
@@ -51,53 +54,63 @@ module.exports = {
-        'base-mobile':  ['0.938rem', '1.375rem'],
-        'heading': fontSize(22, 24),
+        'h1-mobile': ['1.75rem', '2.125rem'],
+        h1: ['3.5rem', '4.125rem'],
+        'h2-mobile': ['1.625rem', '2rem'],
+        h2: ['2.375rem', '2.875rem'],
+        'h3-mobile': ['1.375rem', '1.625rem'],
+        h3: ['1.75rem', '2.25rem'],
+        'h4-mobile': ['1.125rem', '1.438rem'],
+        h4: ['1.5rem', '1.875rem'],
+        'h5-mobile': ['0.938rem', '1.375rem'],
+        h5: ['1.188rem', '1.5rem'],
+        'base-mobile': ['0.938rem', '1.375rem'],
+        heading: fontSize(22, 24),
         'default-base': ['1rem', '1.25rem'],
-        'grande':       ['1.25rem', '1.625rem'],
-        'legendes':     ['0.813rem', '1.25rem'],
+        grande: ['1.25rem', '1.625rem'],
+        legendes: ['0.813rem', '1.25rem'],
       },
       fontFamily: {
-        base:  ['Montserrat', 'Helvetica', 'Arial', 'sans-serif'],
+        base: ['Montserrat', 'Helvetica', 'Arial', 'sans-serif'],
+      },
+      margin: {
+        15: rem(15 * 4),
+      },
+      padding: {
+        2.5: rem(2.5 * 4),
       },
       width: {
         '8.5/12': '70%',
       },
       minWidth: {
+        '8': rem(8 * 4),
       },
       lineHeight: {
-        '12': '3rem',
+        12: rem(12 * 4),
       },
       backgroundSize: {
-        'full': '100%',
-      },
-      spacing: {
+        full: '100%',
       },
       textDecoration: ['focus-visible'],
       boxShadow: {
-        card    : '0px 8.01379px 14.0241px rgba(166, 170, 172, 0.035), 0px 2.89843px 5.07226px rgba(166, 170, 172, 0.0243888)',
-        footer  : '0 0px 5px -10px rgba(0, 0, 0, 0.04), 0 6px 12px -10px rgba(0, 0, 0, 0.05)',
-        toast   : '0px 9px 42px rgba(102, 116, 137, 0.05), 0px 3.75998px 17.5466px rgba(102, 116, 137, 0.0359427), 0px 2.01027px 9.38125px rgba(102, 116, 137, 0.0298054), 0px 1.12694px 5.25905px rgba(102, 116, 137, 0.025), 0px 0.598509px 2.79304px rgba(102, 116, 137, 0.0201946), 0px 0.249053px 1.16225px rgba(102, 116, 137, 0.0140573)',
+        card: '0px 8.01379px 14.0241px rgba(166, 170, 172, 0.035), 0px 2.89843px 5.07226px rgba(166, 170, 172, 0.0243888)',
+        footer: '0 0px 5px -10px rgba(0, 0, 0, 0.04), 0 6px 12px -10px rgba(0, 0, 0, 0.05)',
+        toast: '0px 9px 42px rgba(102, 116, 137, 0.05), 0px 3.75998px 17.5466px rgba(102, 116, 137, 0.0359427), 0px 2.01027px 9.38125px rgba(102, 116, 137, 0.0298054), 0px 1.12694px 5.25905px rgba(102, 116, 137, 0.025), 0px 0.598509px 2.79304px rgba(102, 116, 137, 0.0201946), 0px 0.249053px 1.16225px rgba(102, 116, 137, 0.0140573)',
       },
     },
     screens: {
-      xs      : '576px',
-      sm      : '667px',
-      md      : '768px',
-      lg      : '992px',
-      xl      : '1200px',
-      '2xl'   : '1440px',
-      '3xl'   : '1600px',
+      xs: '576px',
+      sm: '667px',
+      md: '768px',
+      lg: '992px',
+      xl: '1200px',
+      '2xl': '1440px',
+      '3xl': '1600px',
+    },
+    listStyleType: {
```

Comment:
Why do we need this config here? This is a default tailwind rule and you don't change anything in this particular object.